### PR TITLE
session: fix upgrade logic when bootstrap

### PR DESCRIPTION
**Pull request by tangenta**
### What problem does this PR solve? 
Fix upgrade logic when bootstrap.

### Check List 

Tests 

 - Unit test
 - Integration test

Code changes

Side effects

Related changes

Release note

 - Fix upgrade logic when bootstrap.

### CODE REVIEW

File: session/bootstrap.go [hunk 1]
```diff
@@ -520,7 +520,7 @@ func upgrade(s Session) {
 		upgradeToVer28(s)
 	}
 
-	if ver == version28 {
+	if ver < version29 {
```
Comment (shenli) [inline session/bootstrap.go]: upgradeToVer29 only need to be run when the current version is 28

File: session/bootstrap.go [hunk 2]
```diff
@@ -535,7 +536,7 @@ func upgrade(s Session) {
 	}
 
 	if ver < version32 {
-		upgradeToVer29(s)
+		upgradeToVer32(s)
```
Comment (zz-jason) [inline session/bootstrap.go]: could you add some test if it's possible?

Comment (tangenta) [reply session/bootstrap.go]: @zz-jason Do you mean to add a test for unused functions, or something else? I don't know how to test the cross-version issues on a specific version.

**Comment by zz-jason** [comment]
@tangenta do we need to cherry pick this commit to release branches?

**Comment by tangenta** [comment]
> @tangenta do we need to cherry pick this commit to release branches?

@zz-jason I will cherry pick it later.

**Comment by tangenta** [comment]
There is no `version32` on the branch `release-2.1`, so I don't cherry-pick it for now.

**Pull request merged**